modules/marsmediaBidAdapter.js

```diff
@@ -252,12 +252,26 @@ function MarsmediaAdapter() {
       };
 
       if (bidRequest.mediaTypes && bidRequest.mediaTypes.video) {
-        bidResponse.vastUrl = bid.adm;
+        if (bid.adm.charAt(0) === '<') {
+          bidResponse.vastXml = bid.adm;
+        } else {
+          bidResponse.vastUrl = bid.adm;
+        }
         bidResponse.mediaType = 'video';
         bidResponse.ttl = 600;
       } else {
         bidResponse.ad = bid.adm;
       }
+
+      if (!bidResponse.width || !bidResponse.height) {
+        bidResponse.width = 1;
+        bidResponse.height = 1;
+        if (bidRequest.sizes) {
```

Comment:
This should refer to bidRequest.mediaTypes.banner.sizes instead of bidRequest.sizes.